homeassistant/components/plex/media_player.py

```diff
@@ -611,3 +615,84 @@ def device_info(self):
             "sw_version": self._device_version,
             "via_device": (PLEX_DOMAIN, self.plex_server.machine_identifier),
         }
+
+    async def async_browse_media(self, media_content_type=None, media_content_id=None):
+        """Implement the websocket media browsing helper."""
+
+        def item_payload(item):
```

Comment:
These functions can be standalone functions and don't need to be re-defined on each invocation of the browse media function?